extensions/wikia/AdEngine/js/recovery/message.js

```diff
@@ -68,10 +81,10 @@ define('ext.wikia.adEngine.recovery.message', [
 		});
 	}
 
-	function createMessage(uniqueClassName) {
+	function createMessage(uniqueClassName, messageVariant) {
 		return getAssets().then(function (assets) {
 			var template = assets.mustache[0],
-				text = mw.message('adengine-recovery-message-blocking-message-a').rawParams([
+				text = mw.message('adengine-recovery-message-blocking-message-'+messageVariant).rawParams([
```

Comment:
What do you think about a comment above the line `//adengine-recovery-message-blocking-message-a`?